velox/functions/sparksql/DecimalArithmetic.cpp

```diff
@@ -672,6 +672,29 @@ void registerDecimalDivide(
       LongDecimal<P1, S1>,
       ShortDecimal<P2, S2>>({functionName}, constraints);
 }
+
+template <typename TExec>
+struct DecimalCeilFunction {
```

Comment:
The DecimalArithmetic.cpp file is already very large. Might be better to add separate files to implement this function.

Reply:
Moved this to separate files, thanks.